airflow/www/views.py

```diff
@@ -3409,6 +3410,7 @@ def historical_metrics_data(self):
     @expose("/object/next_run_datasets/<string:dag_id>")
     @auth.has_access_dag("GET", DagAccessEntity.RUN)
     @auth.has_access_dataset("GET")
+    @mark_fastapi_migration_done
```

Comment:
We should note this in some contributor docs: https://github.com/apache/airflow//blob/apache-main/contributing-docs/17_adding_api_endpoints.rst

Reply:
Good idea!

Reply:
I added a quick note at the bottom of this document but I think i'll follow up with a complete re-write because adding new endpoints to `api_connexion` / `views.py` is not relevant anymore. The process should instead highlight how to do so in the `FastAPI` context and how to migrate legacy endpoints.

Reply:
https://github.com/apache/airflow/pull/42216

The actual rewrite of the contribution page for adding new endpoints.